Posit Infrastructure/FAQs.md

```diff
@@ -56,6 +57,39 @@ Microsoft Edge puts your tabs to sleep when you're not using them. Tabs are put
 
 ### Sessions
 
+#### How do I find my session ID?
+
+##### Background
+
+When a Posit Workbench session is requsted by the user, it is assigned a session ID.  This is a long string of letters and numbers that uniquely identifies the session. The session ID corresponds to metrics and logs that are produced by the Posit Workbench application and collected for onward analysis and debugging of issues.
+
+##### Steps to find out the session ID of your Posit Workbench session
+
+1. If you have not already done so, log into Posit Workbench and start a session (refer to the [guidance on the Knowledge base here](https://public-health-scotland.github.io/knowledge-base/docs/Posit%20Infrastructure?doc=How%20to%20Access%20Posit%20Workbench.md#logging-in) if necessary).
+
+2. After the session has started, navigate to the [Posit Workbench homepage](https://pwb.publichealthscotland.org) and identify the session you would like the session ID of in the list of sessions, as circled in red below:
```

Comment:
maybe something like "identify the session for which you would like to obtain the session ID" for readability?